pkgs/by-name/sy/sydbox/package.nix

```diff
@@ -1,77 +1,89 @@
-{ lib
-, stdenv
-, fetchurl
-, pkg-config
-, autoreconfHook
-, python3
-, perl
-, libxslt
-, docbook_xsl
-, docbook_xml_dtd_42
-, libseccomp
-, installTests ? true, gnumake, which
-, debugBuild ? false, libunwind
+{
+  lib,
+  fetchFromGitLab,
+  libseccomp,
+  mandoc,
+  nix-update-script,
+  pkg-config,
+  rustPlatform,
+  scdoc,
+  sydbox,
+  testers,
 }:
 
-stdenv.mkDerivation rec {
-  pname = "sydbox-1";
-  version = "2.2.0";
+rustPlatform.buildRustPackage rec {
+  pname = "sydbox";
+  version = "3.28.3";
 
-  outputs = [ "out" "dev" "man" "doc" ]
-    ++ lib.optional installTests "installedTests";
+  outputs = [
+    "out"
+    "man"
+  ];
 
-  src = fetchurl {
-    url = "https://git.exherbo.org/${pname}.git/snapshot/${pname}-${version}.tar.xz";
-    sha256 = "0664myrrzbvsw73q5b7cqwgv4hl9a7vkm642s1r96gaxm16jk0z7";
+  src = fetchFromGitLab {
+    domain = "gitlab.exherbo.org";
+    owner = "Sydbox";
+    repo = "sydbox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-9IegNFkOWYt+jdpN0rk4S/qyD/NSPaSqmFnMmCl/3Tk=";
```

Comment:
_Note:_ There is a mirror at <https://git.sr.ht/~alip/syd>. Just in case `gitlab.exherbo.org` should ever disappear.